examples/with-next/pages/about.js

```diff
@@ -1,7 +1,10 @@
 import { jsx } from "../../../dist/react"
+import Link from 'next/link'
 
 function About() {
-  return <div className="font-bold text-primary">Welcome to About</div>
+  return <div className="font-bold text-blue-600">Welcome to About <Link href="/">
+  <a className="cursor-pointer font-bold text-red-500">Back Home</a>
+</Link>{' '}</div>
 }
```

Comment:
I'd recommend adding `.prettierrc` to the repo since you don't have it enabled by default in VS Code :D